### PR TITLE
[EVENT] GAMESHOW TESTMERGE

### PR DESCRIPTION
**Pull request by CitrusGender**
WE ARE FINALLY ALMOST DONEEEEEEE. Need this PR to testmerge on the server, will likely close this in the future.

(I like having the TM open since I can find it in the list easier, don't worry we'll close this probably by the end of today.)

Input is appreciated, definitely.

DO NOT MERGE

### CODE REVIEW

**Comment by tralezab** [comment]
first for pogging out of my gourd

File: code/game/objects/effects/decals/cleanable.dm [hunk 1]
```diff
@@ -19,7 +19,13 @@
 			if(C != src && C.type == type && !QDELETED(C))
 				if (replace_decal(C))
 					return INITIALIZE_HINT_QDEL
-
+	#ifdef EVENTMODE
+	///AUTO CLEAN
+	if(mapload)
+		return .
+	QDEL_IN(src, 30)
+	return . //skip disease handling
```
Comment (Ghommie) [inline code/game/objects/effects/decals/cleanable.dm:27]: Doesn't `return` already return `.` by default without it being stated?

File: code/game/objects/items/trash.dm [hunk 2]
```diff
@@ -9,12 +9,22 @@
 	item_flags = NOBLUDGEON
 
 /obj/item/trash/Initialize(mapload)
+	#ifdef EVENTMODE
+	///AUTO CLEAN
+	if(mapload)
+		return .
```
Comment (Ghommie) [inline code/game/objects/items/trash.dm:15]: ditto.

**Comment by CitrusGender** [comment]
it's event code

**Comment by CitrusGender** [comment]
changing to actual branch.

**Pull request closed without merging**